client/src/components/RatingStar.jsx

```diff
@@ -0,0 +1,17 @@
+class RatingStar extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <span className='StarBox'>
+        <span className='outerStar'>
+          <span className='innerStar' style={{width: `${this.props.rating * 22}px`}}></span>
```

Comment:
Finally, you figured out! congrats!

Reply:
Thanks. And I just noticed that I have to change the class name to avoid conflict.